x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -1333,6 +1333,8 @@ interface BaseListResponse<D = unknown> {
   page: number;
   pageSize: number;
   total: number;
+  sortField: string;
```

Comment:
I don't remember how many of our types uses this base interface, but if its more than just the Endpoint list, then these two additional properties are now showing up on other types, which I don't think is what we want (for now).  you might need to add them explicitly to the metadata list return type.

Reply:
it's not used elsewhere. so... I guess it's not even a base interface

just removed one level of the unnecessary abstraction:
```ts
export interface MetadataListResponse {
  data: HostInfo[];
  page: number;
  pageSize: number;
  total: number;
  sortField: string;
  sortDirection: 'asc' | 'desc';
}
```

Reply:
ba235c6b59dedeeb7c09225cb257797780563333

Reply:
Yeah, the goal with that "base" type was to ensure that going forward our API responses (new APIs) all get a "standard" interface. This is something I remember discussing during a Eng. Hour meeting a while back. 

I'm ok with the change you did (removed it)